jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -257,6 +261,25 @@ public void testPropagationJaegerAndB3() {
     assertEquals(spanId, extractedContext.getSpanId());
   }
 
+  @Test
+  public void testPropagationBinary() {
+    System.setProperty(Configuration.JAEGER_PROPAGATION, "jaeger");
+    System.setProperty(Configuration.JAEGER_SERVICE_NAME, "Test");
+
+    long traceIdLow = 1234L;
+    long spanId = 5678L;
+
+    TestBinaryCarrier buffer = new TestBinaryCarrier();
+    JaegerSpanContext spanContext = new JaegerSpanContext(0, traceIdLow, spanId, 0, (byte)0);
```

Comment:
Do we have boundary tests here? If so, it would be nice to have them for the binary codec as well.

Reply:
Do you mean boundaries as in max values for trace ids, spanIds, etc?  I didn't see any elsewhere (may have missed them), but can add them to the binary codec if you'd like.

Reply:
Don't worry about this test for now: if we get an ID which is bigger than the max value, it would also fail for other codecs. I was also a bit concerned about parsing failures (baggage keys without values, for instance) , but I believe this first implementation is already a great improvement.